components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/spark/algorithms/SupervisedSparkModelBuilder.java

```diff
@@ -708,7 +733,7 @@ private MulticlassConfusionMatrix getMulticlassConfusionMatrix(MulticlassMetrics
      * @param predictionsAndLabels Prediction and label values RDD
      */
     private RegressionMetrics getRegressionMetrics(JavaSparkContext sparkContext,
-            JavaRDD<Tuple2<Double, Double>> predictionsAndLabels) {
+                                                   JavaRDD<Tuple2<Double, Double>> predictionsAndLabels) {
```

Comment:
Better to send a separate PR for formatting purposes, if you are using WSO2 Code template.